packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroupToolbarItem.tsx

```diff
@@ -18,14 +19,20 @@ export const ChipGroupToolbarItem: React.FunctionComponent<ChipGroupToolbarItemP
   ...props
 }: ChipGroupToolbarItemProps) => {
     if (React.Children.count(children)) {
-      return(
-        <ul className={css(styles.chipGroup, styles.modifiers.toolbar)} {...props}>
-          <li>
-            <h4 className={css(styles.chipGroupLabel)}>{categoryName}</h4>
-            <ul className={css(styles.chipGroup)}>{children}</ul>
-          </li>
-        </ul>
+      return (
+        <ChipGroupContext.Consumer>
+          {(HeadingLevel: any) => {
```

Comment:
it was passed in lowercase, this works? also i looked at the built docs and it doesn't show `headingLevel` prop for ChipGroup, but rather under ChipGroupToolbarItem. Maybe it wasn't rebuilt?

Reply:
Ah just checked, the value is passed directly and then picked up here. So that's good. and the docs looks fine as well when i built it locally, LGTM

Reply:
Hi @jschuler - good questions:

You are correct if chip group context is an object containing the `headingLevel` attribute but since the chip group context is only headingLevel then it's OK that the argument will be `HeadingLevel`.

If you want I can change the context to be object:

```
type ChipGroupCtx = {
  headingLevel: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
}

ChipGroupContext = React.React.createContext<Partial<ChipGroupCtx>>({});
```

And in here I will have to change it to this:

```
{({headingLevel: HeadingLevel}) => (...)
```

Regarding to the props that aren't shown in the docs, I can't look at the CI but I assume it built the docs and uploaded but probably the URL was changed and for some reason it is not shown in here.

Anyway, let me know if I'm missing anything and thanks for your review!